onnxscript/optimizer/_function_folding_test.py

```diff
@@ -60,13 +60,11 @@ def test_sequence_concat(self):
         """
         )
         optimized = optimizer.optimize(
-            model,
-            onnx_shape_inference=False,
-            num_iterations=1,
+            model, onnx_shape_inference=False, num_iterations=1, inline=False
         )
         function_node = optimized.functions[0].node
-        self.assertEqual(len(function_node), 3)
         self.assertEqual(function_node[2].op_type, "Concat")
+        self.assertEqual(len(function_node), 3)
```

Comment:
Why reorder? The length assertion guards the indexing operation, which may be invalid if length < 3.

Reply:
Makes sense. Will revert.